Task_TheGridProblem/index.css

```diff
@@ -1 +1,7 @@
-/* Write code below for GRID PROBLEM TASK */
+/* Write code below for GRID PROBLEM TASK */
+.dd{
+display: grid;
+gap: 1rem;
+grid-template-columns: 1fr 1fr 1fr;

```

Comment:
The grid-template-rows values should add up to 100% for the grid to fill the available space properly.